### DISCUSSION TITLE
purpose of -name in gen_coords?

### DISCUSSION THREAD

**Discussion by csbrasnett** [marrink-lab/polyply_1.0#289] [General]
-name is a required argument in gen_coords, but it is unclear to me as to why at the moment. It doesn't appear to be related to the topology as far as I can see, and it's not being passed to any header information. Is there some functionality that I'm missing?

**Comment by fgrunewald** [comment]
I guess not really! Originally the idea was to have a system name in the gro file plus some complications with the arg parser at the time. But we recently also changed the name requirement for the gen_params program, so we could do the same here I suppose. Feel free to make a PR
